### DISCUSSION TITLE
How to divide apps by Service Groups in Skywalking?

### DISCUSSION THREAD

**Discussion by SergBw** [apache/skywalking#11882] [Q&A]
Hi. My goal is to use SkyWalking as my main monitoring and observability tool for apps in the k8s cluster. In some cases, I have 2 different apps in one cluster, in others - 2 environments of one app in one cluster. I want to gather all their data in one skywalking instance.
How can I divide all these apps into different Service Groups in UI? Or is there any other proper way to do it?
Thanks in advance!

**Comment by wu-sheng** [comment]
Which kinds of monitoring stack are you using? Group usually could be defined by yourself.

**Reply by SergBw** [reply]
Go and node agents, Istio mesh, and k8s cluster

**Reply by wu-sheng** [reply]
Those are in different ways. 
1. Service name of the agents could be set manually.
2. Mesh(ALS based) are using `metadata-service-mapping.yaml`
3. K8s cluster is using MAL analysis scripts, https://github.com/apache/skywalking/tree/master/oap-server/server-starter/src/main/resources/otel-rules/k8s, especially, https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/otel-rules/k8s/k8s-service.yaml#L32

Basically, all of them are following this rule, https://skywalking.apache.org/docs/main/v9.7.0/en/setup/backend/service-auto-grouping/#service-auto-grouping
